vllm/engine/metrics.py

```diff
@@ -331,6 +331,15 @@ def log(self, stats: Stats) -> None:
                 generation_throughput=generation_throughput)
 
             # Log to stdout.
```

Comment:
This comment should be removed since this module has no idea how the logger may be configured.